include/rocksdb/advanced_options.h

```diff
@@ -668,6 +670,8 @@ struct AdvancedColumnFamilyOptions {
   // Env).
   //
   // Only supported in Level compaction.
```

Comment:
This line can now be updated to something like "Supported in Level and FIFO compactions."